### ISSUE TITLE
Standardize Border-Radii

### ISSUE THREAD

**Issue by cikeddy** [HealthCatalyst/Fabric.Cashmere#2012]
**Summary**
Our modals, chips, buttons, and tooltips have different a border-radius.

**Reproduction**
Modal=6px, Button=5px, Chip= 6px, Beta Chip in Banner=4px, Instance Switcher Chip = 6px, Environment Switcher Chip = 4px, Popover/Tooltip = 4px

**Comment by corykon** [comment]
Related: https://github.com/HealthCatalyst/Fabric.Cashmere/issues/2003

**Comment by corykon** [comment]
I wonder if in _some_ cases different sizes are a good thing - either to communicate different functionality, or, just because different size border radii read differently on different size container. Kind of like how you may want to change kerning or line height at different font sizes.

Worth reviewing regardless :+1:

**Comment by cikeddy** [comment]
I agree! I think for things that are smaller we should probably decrease the border radius because it looks weird if you don't. I think for larger things we should also increase the border radius to maintain the look and feel.  For example: 
![image](https://user-images.githubusercontent.com/46426253/215840724-028dfdba-7da1-434b-9de2-94860ec9a476.png)
